Dockerfile

```diff
@@ -1,9 +1,15 @@
 ARG DEBIAN_VERSION=12.8
+ARG EPICS_IN_DOCKER_VERSION=v0.12.0-dev
 
-FROM ghcr.io/cnpem/lnls-debian-epics-7:v0.12.0-dev AS build-image
+FROM ghcr.io/cnpem/lnls-debian-epics-7:${EPICS_IN_DOCKER_VERSION} AS build-image
```

Comment:
We might have to repeat these values in two places... The problem with making `EPICS_IN_DOCKER_VERSION` an argument is that it can be overridden by the `docker-compose.yml` definitions, which we don't want. We want the Dockerfile versions to be matched to the base image versions.